spec/order_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'order'
+
+describe Order do
+
+  let(:new_dish) { {name: "pizza", quantity: 2, price: 14, accumulated_prize: 28} }
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)